### PR TITLE
Tabular: Minor improvements

### PR DESCRIPTION
**Pull request by Innixma**
*Issue #, if available:*

*Description of changes:*

- GBM preset -> GBMLarge and moved to 'GBM' list so it is filtered out by exclude_models.
- Documented 'multimodal' hyperparameters preset.
- Minor cleanup on TextPredictor.predict.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-964-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-964/2/index.html

File: tabular/src/autogluon/tabular/predictor/predictor.py [hunk 1]
```diff
@@ -299,11 +300,13 @@ def fit(self,
...
-                Valid `str` options: ['default', 'light', 'very_light', 'toy']
+                Valid `str` options: ['default', 'light', 'very_light', 'toy', 'multimodal']
                     'default': Default AutoGluon hyperparameters intended to maximize accuracy without significant regard to inference time or disk usage.
                     'light': Results in smaller models. Generally will make inference speed much faster and disk usage much lower, but with worse accuracy.
                     'very_light': Results in much smaller models. Behaves similarly to 'light', but in many cases with over 10x less disk usage and a further reduction in accuracy.
                     'toy': Results in extremely small models. Only use this when prototyping, as the model quality will be severely reduced.
+                    'multimodal': [EXPERIMENTAL] Trains a multimodal transformer model alongside tabular models. Requires text columns, a GPU, and CUDA-enabled MXNet.
+                        When combined with 'best_quality' `presets` option, this can achieve extremely strong results in multimodal datasets.
```
Comment (jwmueller) [inline tabular/src/autogluon/tabular/predictor/predictor.py:309]: results in multimodal data tables that contain columns with text in additional to numeric/categorical columns.

File: tabular/src/autogluon/tabular/predictor/predictor.py [hunk 2]
```diff
@@ -299,11 +300,13 @@ def fit(self,
         hyperparameters : str or dict, default = 'default'
             Determines the hyperparameters used by the models.
             If `str` is passed, will use a preset hyperparameter configuration.
-                Valid `str` options: ['default', 'light', 'very_light', 'toy']
+                Valid `str` options: ['default', 'light', 'very_light', 'toy', 'multimodal']
                     'default': Default AutoGluon hyperparameters intended to maximize accuracy without significant regard to inference time or disk usage.
                     'light': Results in smaller models. Generally will make inference speed much faster and disk usage much lower, but with worse accuracy.
                     'very_light': Results in much smaller models. Behaves similarly to 'light', but in many cases with over 10x less disk usage and a further reduction in accuracy.
                     'toy': Results in extremely small models. Only use this when prototyping, as the model quality will be severely reduced.
+                    'multimodal': [EXPERIMENTAL] Trains a multimodal transformer model alongside tabular models. Requires text columns, a GPU, and CUDA-enabled MXNet.
```
Comment (jwmueller) [inline tabular/src/autogluon/tabular/predictor/predictor.py:308]: "Requires text columns"
-> "Requires that some text columns appear in the data"

**Review by jwmueller — APPROVED** [review]

File: tabular/src/autogluon/tabular/predictor/predictor.py (hunk 1 above)
Comment (Innixma) [reply tabular/src/autogluon/tabular/predictor/predictor.py:309]: Added

File: tabular/src/autogluon/tabular/predictor/predictor.py (hunk 2 above)
Comment (Innixma) [reply tabular/src/autogluon/tabular/predictor/predictor.py:308]: Added

**Comment by szha** [comment]
Job PR-964-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-964/4/index.html

**Pull request merged**